### DISCUSSION TITLE
Setting up Retargeting Pixel

### DISCUSSION THREAD

**Discussion by mcnaveen** [YOURLS/YOURLS#3082] [General]
Is there any way I can use Retargeting Pixel in Yourls? Example: Facebook etc.,

**Comment by ozh** [comment]
Yes. Example: Plugins etc.,

**Reply by mcnaveen** [reply]
Oh. Is there any plugin available for that already?

**Reply by ozh** [reply]
https://github.com/YOURLS/awesome-yourls
